nanoFramework.Device.Bluetooth/INativeDevice.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace nanoFramework.Device.Bluetooth
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public interface INativeDevice
+    {
+        /// <summary>
+        /// Initializes the Gatt service.
+        /// </summary>
+        void InitService();
+        
+        /// <summary>
+        /// Starts Advertising the Gatt service
+        /// </summary>
+        /// <returns>if advertising started correctly.</returns>
+        bool StartAdvertising();
+        
+        /// <summary>
+        /// Stops advertising the Gatt service
+        /// </summary>
+        void StopAdvertising();
+        
+        /// <summary>
+        /// Sends a notification to a Gatt subscribed client.
+        /// </summary>
+        /// <param name="connection">The device id.</param>
+        /// <param name="characteristicId">The characteristic id.</param>
+        /// <param name="notifyBuffer">The buffer with notification data.</param>
+        /// <returns></returns>
+        public int NotifyClient(ushort connection, ushort characteristicId, byte[] notifyBuffer);
```

Comment:
Can other packages use an interface if it is anything other than public?

Reply:
The visibility is not determined in the interface but in its implementation. See https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/interface

Reply:
Now, to use the interface somewhere else, yes, the interface itself has to be public otherwise, you can't use it. But you'll get an error trying to hide it if used in a public class. I hope both comments make sense.

Reply:
Yes, agree about that. Just that the visibility in the Interface itself is not needed. It's in the implementation that it takes place.

Reply:
ok I changed that